### PR TITLE
Add Microsoft.Extensions.Configuration.AzureKeyVault to package list

### PR DESCRIPTION
**Pull request by pakrym**
After https://github.com/aspnet/Configuration/pull/500 goes in.
@pranavkm

### CODE REVIEW

**Comment by pranavkm** [comment]
:shipit:

File: packages/packages.csv [hunk 1]
```diff
@@ -133,6 +133,7 @@ Microsoft.Extensions.Configuration.Ini,ship,include
 Microsoft.Extensions.Configuration.Json,ship,include
 Microsoft.Extensions.Configuration.UserSecrets,ship,include
 Microsoft.Extensions.Configuration.Xml,ship,include
+Microsoft.Extensions.Configuration.AzureKeyVault,ship,include
```
Comment (victorhurdugaci) [inline packages/packages.csv]: Is alphabetic sorting different in Russian? :)

Comment (pakrym) [reply packages/packages.csv]: Yeah, `A` is the last letter. :)

**Comment by victorhurdugaci** [comment]
:shipit:

**Pull request merged**